Chart.yaml

```diff
@@ -1,7 +1,7 @@
 # apiVersion v2 is Helm 3
 apiVersion: v2
 name: airflow
-version: 1.0.9
+version: 1.0.10
```

Comment:
Aren't we already on 1.3.0?

Reply:
this is for 0.27 of astronomer

Reply:
Ah, I see the target branch now, thanks.